driver/src/test/java/org/neo4j/driver/integration/ConnectionHandlingIT.java

```diff
@@ -254,7 +254,7 @@ void connectionUsedForTransactionReturnedToThePoolWhenTransactionFailsToCommitte
     {
         try ( Session session = driver.session() )
         {
-            session.run( "CREATE CONSTRAINT ON (book:Library) ASSERT exists(book.isbn)" );
+            session.run( "CREATE CONSTRAINT FOR (book:Library) REQUIRE book.isbn IS NOT NULL" );
```

Comment:
this needs to depend per neo4j version
(this syntax is valid with 4.4 and 5.0 but not earlier)